### DISCUSSION TITLE
Lifetime License has an Expiration Date

### DISCUSSION THREAD

**Discussion by alteredchimera** [spatie/ray#682] [General]
@freekmurze 

Just curious why a Lifetime License has a Expiration Date?

![Screen Shot 2022-04-26 at 12 18 08 PM](https://user-images.githubusercontent.com/87214300/165346615-cc8ee858-7ab5-4551-b02b-114a2774d56a.png)

Is it a system whereby you will keep extending the Expiration date as it drawers close... I totally plan to be using Ray beyond that expiration date! Lol. 

Thanks

**Comment by freekmurze** [comment]
We'll be sure to fix that when we approach 2038 at the latest 😅

**Comment by alteredchimera** [comment]
I appreciate the response haha. Thanks for a stellar sidekick (Ray) and thanks for all your awesome work.
